modules/core/src/main/java/org/apache/ignite/internal/wrapper/Wrappers.java

```diff
@@ -34,4 +36,19 @@ public static <T> T unwrap(Object object, Class<T> classToUnwrap) {
 
         return classToUnwrap.cast(object);
     }
+
+    /**
+     * Unwraps an object or returns {@code null} if it's {@code null}.
+     *
+     * @param object Object to unwrap.
+     * @param classToUnwrap Class which is to be unwrapped.
+     * @see #unwrap(Object, Class)
+     */
+    public static @Nullable <T> T unwrapNullable(@Nullable Object object, Class<T> classToUnwrap) {
```

Comment:
I saw that method `java.lang.Class#cast` will return `null` if `null` is passed to it, I think we can change the signature of `org.apache.ignite.internal.wrapper.Wrappers#unwrap` or add a check to it.